test/test_tensor_creation_ops.py

```diff
@@ -751,7 +756,7 @@ def test_cat_out_memory_format(self, device):
         res2_cpu = torch.cat((a_cpu, b_cpu), out=out_cpu)
 
         self.assertTrue(res2_cuda.is_contiguous(memory_format=torch.contiguous_format))
-        self.assertTrue(res2_cpu.is_contiguous(memory_format=torch.channels_last))
+        self.assertTrue(res2_cpu.is_contiguous(memory_format=torch.contiguous_format))
```

Comment:
Since I'm using only one method for inferring the memory format, CPU and CUDA behave consistently.